fretwell-distributed-government.md

```diff
@@ -1,98 +1,93 @@
 
-* Sustainability: Decreased carbon footprint as fewer people are commuting.
+Here are just a few:
 
-* Work/life balance: Government employees spend less time commuting and invest more energy into their families and communities.
+* Remote workers earn salaries higher than $100,000/year, 2.2 times more frequently than on-site workers.
 
-* Economic empowerment: Financial employment benefits are distributed across multiple local communities rather than centralizing to one.
+* Remote workers say they're happy in their jobs 29% more than on-site workers; 71% of remote workers say they're happy in their job, and only 55% of on-site workers say they're happy in their job.
 
-## Increased empathy
+* 34% of U.S. workers would take a pay cut of up to 5% in order to work remotely.
 
-In addition to the above reasons, distributed government teams can tap into a stronger sense of empathy — a value inherent in the role of authentic public service.
+* 42% of remote workers plan to work remotely more frequently than they currently do in the next five years, and that more than half of on-site workers want to start working remotely.
 
-Once you truly experience distributed culture, you have a stronger understanding of what digital services really mean and how they can make or break the end user experience. If public servants don't place themselves in digital environments of their own, their ability to have complete empathy for those they serve is lessened.
+* 55% of remote workers would be likely to look for another job if they were no longer allowed to work remotely. And 61% of remote workers would expect a pay increase if they were no longer allowed to work remotely.
 
-Distributed digital government service teams have more potential to have empathy for their end users. Immersing yourself into a distributed team is the ultimate digital service user research experience.
+* 68% of remote workers say they are not concerned working remotely will impact their career progression, while 23% say they fear it would.
 
-## We have the means
+* The top reasons remote workers choose to work remotely include: better work-life balance (91%), increased productivity/better focus (79%), less stress (78%) and avoiding a commute (78%).
 
-We now have low-cost, high-reliability tools — G Suite, Slack, Trello, GitHub and Zoom to name just a few — that fully empower asynchronous, instant collaboration. Training on the tools is of course important. Just as important is the training on how to work in and lead distributed teams, fostering a culture of distributed work, and implementing policies to support highly effective distributed teams.
+* Remote workers say they work more than 40 hours per week 43% more than on-site workers do. However, on-site workers are also working longer weeks because it's required of them, while more remote workers are doing so because they enjoy what they do.
 
-## Government is doing this
+_XXX_
```

Comment:
Placeholder for additional material: Can you offer any kind of summarizing thought/perspective for this section, something that bridges to where you'd like readers to follow you next? As it stands, the section just sort of ends abruptly with a string of statistics. What should we make of them? What do they suggest viz. distributed government?